lib/tls.js

```diff
@@ -15,6 +15,13 @@ exports.CLIENT_RENEG_WINDOW = 600;
 exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
 exports.DEFAULT_CIPHERS = [
+  'ECDHE-RSA-AES128-GCM-SHA256',
+  'ECDHE-ECDSA-AES128-GCM-SHA256',
+  'ECDHE-RSA-AES256-GCM-SHA384',
+  'ECDHE-ECDSA-AES256-GCM-SHA384',
+  'DHE-RSA-AES128-GCM-SHA256',
+  'DHE-DSS-AES128-GCM-SHA256',
+  'kEDH+AESGCM',
```

Comment:
Re: 'is this one necessary?'

Do you mean the whole six lines with the GCM ciphers? Yes, that's necessary. **Most things which don't explicitly have GCM are CBC**. You can prove this and convert the custom names OpenSSL uses to the ones Chrome and SSL Labs use here: https://www.openssl.org/docs/apps/ciphers.html